src/components/nodes-table/ColumnFilterDate.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <v-card-text>
+    <v-row>
+      <v-col>
+        <v-text-field
+          type="datetime-local"
+          label="From"
+          v-model="value.from"
+          clearable
+          @change="$emit('change', value)"
+        ></v-text-field>
+        <v-text-field
+          type="datetime-local"
```

Comment:
This should not allow set a date that is before the `value.from`

Reply:
Done.